django_tasks/__init__.py

```diff
@@ -1,3 +1,8 @@
+# ruff: noqa: E402
```

Comment:
Issue: Could this be ignored for the specific lines, rather than the entire file.

Reply:
We would need to ignore it on EVERY import that follow this line. That looked kind of ugly. Would you prefer having it on every following import?

Reply:
Oh, good point - that'd be a problem. This sure isn't great, but I can't think of anything better!